### DISCUSSION TITLE
Authenticate using Postman to test REST API

### DISCUSSION THREAD

**Discussion by rvishna** [coder/code-server#2915] [Q&A] [answered]
I am using code-server to develop a REST API. I am running code-server with the PASSWORD environment variable (not --link mode).

```
$ code-server --version
3.9.1 e0203f2a36c9b7036fefa50eec6cf8fa36c5c015
```

With static HTML pages, I've been able to use the /proxy/<port> or /absproxy/<port> suffix to access them from the browser after authenticating to code-server. For testing my REST API, I am not sure how to authenticate using Postman. I am testing a POST request. My REST API itself doesn't require authentication. But since it is being proxied through code-server, I am not sure how to authenticate.

**Comment by code-asher** [comment, marked as answer]
Looks like Postman supports cookies (https://learning.postman.com/docs/sending-requests/cookies/) so you could add a cookie with the name of `key` set to the value of the sha256 of your password (you can either hash it yourself or copy it from the browser).
